girder/wsgi.py

```diff
@@ -18,14 +18,24 @@
 ###############################################################################
 
 import cherrypy
-from girder.utility import config, server
+import girder
+import logging.handlers
+from girder.utility import server
 
-config.loadConfig()  # Read the config files first, so we can override some values
 cherrypy.config.update({'engine.autoreload.on': False,
                         'environment': 'embedded'})
 cherrypy.config['server'].update({'cherrypy_server': False,
                                   'disable_event_daemon': True})
 
+# TODO The below line can be removed if we do away with girder.logprint
+girder._quiet = True  # This means we won't duplicate messages to stdout/stderr
+_formatter = girder.LogFormatter('[%(asctime)s] %(levelname)s: %(message)s')
```

Comment:
Is it possible or desirable to follow the [Common Log Format](https://en.wikipedia.org/wiki/Common_Log_Format) here?

Reply:
I think that's a good idea. Would it be too disruptive to also change it in the main girder server context?

Reply:
Actually it looks like that format only covers access logs, i.e. basic request info, not general purpose logging. I vote we leave formatting changes for a future branch.